ui/src/main/js/store/actions/jira-server.js

```diff
@@ -0,0 +1,332 @@
+        }
+        saveRequest.then((response) => {
+            if (response.ok) {
+                dispatch(saveJiraServerSuccess());
+            } else {
+                response.json()
+                    .then((responseData) => {
+                        const defaultHandler = () => saveJiraServerFail(responseData);
+                        errorHandlers.push(HTTPErrorUtils.createBadRequestHandler(defaultHandler));
+                        errorHandlers.push(HTTPErrorUtils.createDefaultHandler(defaultHandler));
+                        const handler = HTTPErrorUtils.createHttpErrorHandler(errorHandlers);
+                        dispatch(handler(response.status));
+                    });
+            }
+        })
+            .catch(console.error);
+    };
+}
+
+export function deleteJiraServer(jiraServerModels) {
+    return (dispatch, getState) => {
+        dispatch(deleteJiraServerRequest());
+        const { csrfToken } = getState().session;
+
+        Promise.all(jiraServerModels.map((server) => { // eslint-disable-line
+            return ConfigRequestBuilder.createDeleteRequest(JIRA_SERVER_URLS.jiraServerConfigUrl, csrfToken, server.id);
+        })).catch((error) => {
+            dispatch(deleteJiraServerError(error));
+            console.error; // eslint-disable-line
+        }).then((response) => {
+            if (response) {
+                dispatch(deleteJiraServerSuccess());
+            }
+        });
+    };
+}
+
+export function testJiraServer(jiraServerModel) {
+    return (dispatch, getState) => {
+        dispatch(testJiraServerRequest());
+        const { csrfToken } = getState().session;
+        const errorHandlers = [];
+        errorHandlers.push(HTTPErrorUtils.createUnauthorizedHandler(unauthorized));
+        errorHandlers.push(HTTPErrorUtils.createForbiddenHandler(() => testJiraServerrFail(HTTPErrorUtils.MESSAGES.FORBIDDEN_ACTION, {})));
+        const testRequest = ConfigRequestBuilder.createTestRequest(JIRA_SERVER_URLS.jiraServerConfigUrl, csrfToken, jiraServerModel);
+        testRequest.then((response) => {
+            if (response.ok) {
+                response.json()
+                    .then((testResponse) => {
+                        if (testResponse.hasErrors) {
+                            handleValidationError(dispatch, errorHandlers, response.status, () => testJiraServerrFail(testResponse.message, testResponse.errors));
+                        } else {
+                            dispatch(testJiraServerSuccess());
+                        }
+                    });
+            } else {
+                handleValidationError(dispatch, errorHandlers, response.status, () => testJiraServerrFail(response.message, response.errors));
+            }
+        })
+            .catch(console.error);
+    };
+}
+
+export function sendJiraServerPlugin(azureBoard) {
```

Comment:
"azureBoard" seems to be included here by mistake

Reply:
Good catch! I'll have this updated in the next commit.